scripts/HoveredMidiRelative/HoveredMidiRelativeExt.py

```
@@ -516,119 +626,164 @@ def onReceiveMidiSlotLearn(self, index: int):
+					self.activePar.max = _val
+					
+					# Create undo action
+					self.undo_manager.create_set_norm_undo(
+						self.activePar, is_min=False,
+						old_norm=old_norm, new_norm=_val,
+						old_minmax=old_minmax, new_minmax=_val
+					)
+			
+			# update display
+			self.display_manager.update_parameter_display(self.activePar, bottom_text=f'_{min_max.upper()}_')
+		except:
```

Ruff (0.14.3)

Do not use bare `except`

(E722)